pkg/database/orchestrator/client/client.go

```diff
@@ -62,6 +63,26 @@ func New(host string) *orcClient {
 	}
 }
 
+func (o *orcClient) Discover(host string, port int32) (*instance, error) {
```

Comment:
I feel that we agreed that we will add only pod-0 into the Orchestrator (via config or Startup probe).

Reply:
please see the PR description:

> With these changes, we discover pods by sending requests to Orchestrator
from the operator itself. Unfortunately, the assumption that Orchestrator
will discover every instance if we give it only one host is wrong. So,
we need to send a request for every MySQL pod.